starknet_devnet/udc.py

```diff
@@ -44,3 +46,10 @@ def contract_class(self) -> CompiledClassBase:
 
     async def _mimic_constructor(self):
         pass
+
+    def _print(self):
+        """stdout UDC"""
```

Comment:
If the parent `_print` has a docstring, the children don't need it (especially if it's not really informative). But then the parent can have a more verbose docstring.

Reply:
Thx!